exercises/concept/amusement-park-improvements/.docs/instructions.md

```diff
@@ -14,6 +14,8 @@ Attendee.new(100).has_pass?
 ## 2. Check if an attendee fits a ride
 
 Implement the `Attendee#fits_ride?` method to see if an attendee fits a ride based on their height.
+The ride's required minimum height is provided as an argument.
```

Comment:
Does the student know what an "argument" is at this stage in the track?

Reply:
Reply by e-mail apparently does not link very well.

Reply:
On 22/01/19 11:44AM, Jeremy Walker wrote:
> @iHiD commented on this pull request.
> 
> 
> 
> > @@ -14,6 +14,8 @@ Attendee.new(100).has_pass?
>  ## 2. Check if an attendee fits a ride
>  
>  Implement the `Attendee#fits_ride?` method to see if an attendee fits a ride based on their height.
> +The ride's required minimum height is provided as an argument.
> 
> Does the student know what an "argument" is at this stage in the track?
> 
They should, as they would have also solved the Amusement Park which
takes the arguments.  But I will read the exercise to make sure.

As the instructions for the earlier exercise states:

> # Instructions
> 
> Working with an amusement park, you've been handed a specification to
> design a system to administer attendance and rides. You've been tasked
> with modeling the Attendee (person visiting the park).
> 
> ## 1. Make new attendees
> 
> Implement the `Attendee#initialize` method of the `Attendee` class, it
> should take a height (in centimeters) and store it as an instance
> variable

I think "should take a height" covers it, though not by name.